src/scripts/main.js

```diff
@@ -1,3 +1,20 @@
 'use strict';
 
-// write your code here
+const ArrayOfSelectorPopulation = [...document.querySelectorAll('.population')];
```

Comment:
```suggestion
const arrayOfSelectorPopulation = [...document.querySelectorAll('.population')];
```
name vars and functions in camelCase